test/init.js

```diff
@@ -5,236 +5,262 @@
+    __dirname,
+    "../bundles/dna/planning/hrea_planning.dna"
+  ),
+  proposal: path.resolve(
+    __dirname,
+    "../bundles/dna/proposal/hrea_proposal.dna"
+  ),
+  specification: path.resolve(
+    __dirname,
+    "../bundles/dna/specification/hrea_specification.dna"
   ),
-})
+  plan: path.resolve(__dirname, "../bundles/dna/plan/hrea_plan.dna"),
+};
+const getDNA = (name) => dnaPaths[name];
 
 /**
  * Create per-agent interfaces to the DNA
  */
 const buildGraphQL = async (player, apiOptions, appCellMapping) => {
-
-  const tester = new GQLTester(schema, resolverLoggerMiddleware()(await generateResolvers({
-    ...apiOptions,
-    conductorUri: player.appWs().client.socket._url,
-    dnaConfig: appCellMapping,
-    traceAppSignals: (signal) => {
-      console.info('App signal received:', signal)
-    },
-  })))
+  const tester = new GQLTester(
+    schema,
+    resolverLoggerMiddleware()(
+      await generateResolvers({
+        ...apiOptions,
+        conductorUri: player.conductor.appWs().client.socket._url,
+        dnaConfig: appCellMapping,
+        traceAppSignals: (signal) => {
+          console.info("App signal received:", signal);
+        },
+      })
+    )
+  );
 
   return async (query, params) => {
     const result = await tester.graphql(query, undefined, undefined, params);
 
     // GraphQL errors don't get caught internally by resolverLoggerMiddleware, need to be printed separately
     (result.errors || [])
-      .filter(err => err instanceof GraphQLError)
-      .forEach(e => {
-        console.warn('\x1b[1m\x1b[31mGraphQL query error\x1b[0m', e)
-      })
+      .filter((err) => err instanceof GraphQLError)
+      .forEach((e) => {
+        console.warn("\x1b[1m\x1b[31mGraphQL query error\x1b[0m", e);
+      });
 
-    return result
-  }
-}
+    return result;
+  };
+};
 
 /**
  * Creates bindings for a player against a single hApp, returning a GraphQL client
  * as well as the underlying Holochain DNA `cells`.
  */
-const buildPlayer = async (scenario, config, agentDNAs, graphQLAPIOptions) => {
-  const [player] = await scenario.players([config])
-  const agentPubKey = await player.adminWs().generateAgentPubKey()
-  const dnaSources = agentDNAs.map(getDNA)
-  const dnas = await Promise.all(dnaSources.map(async (dnaSource, index) => {
-    const dnaHash = await player.registerDna({ path: dnaSource })
-    return {
-      hash: dnaHash,
-      role_id: agentDNAs[index]
-    }
-  }))
-  const installAppReq = {
-    installed_app_id: 'installed-app-id',
-    agent_key: agentPubKey,
-    dnas: dnas
-  }
-  await player.adminWs().installApp(installAppReq)
-  // must be enabled to be callable
-  const enabledAppResponse = await player.adminWs().enableApp({
-      installed_app_id: installAppReq.installed_app_id
-  })
-  if (enabledAppResponse.errors.length > 0) {
-      throw new Error(`Error - Failed to enable app: ${enabledAppResponse.errors}`)
+const buildPlayer = async (agentDNAs, graphQLAPIOptions) => {
```

Comment:
I also accidentally formatted this file. If I/we can establish a standard formatting method it'd be great.
I personally recommend `prettier`.

Anyways, this function `buildPlayer` is the only thing with logical changes.

Reply:
I'm using https://standardjs.com/ - there is an `.eslintrc` in the `test/` directory with a config, and a `tslint.json` in one of the `modules/` (obviously, should be in both).

Your editor might not be picking these up- if not, feel free to add more dotfiles as needed.

Reply:
Ok I'll see if I can get those working. Using VS Code

Reply:
I got it working. I formatted all files according to those rules

Reply:
I had to update and tweak the .eslintrc file slightly.